programs/uxd/src/test/utils/math/compute_amount_less_fraction.rs

```diff
@@ -29,6 +30,17 @@ mod test_compute_amount_less_fraction {
         // Test proper precision loss behavior
         assert_eq!(compute_amount_less_fraction(1000, 1, 10000)?, 999);
         assert_eq!(compute_amount_less_fraction(1000, 9999, 10000)?, 0);
+
+        // Test large amounts against u64 overflow
+        assert_eq!(
```

Comment:
Can we add a test with `u64::MAX` value please?

Reply:
Yes ser, here we goo:
https://github.com/UXDProtocol/uxd-program/pull/213/commits/058eb6de90ce446e9774081bcf1fae211bfdc029

Reply:
I think we should add prop_tests, they do test on min/max value among random ones

Reply:
Since we are adding a lot of unit test, prop_test is quite easy to setup, and catch all these nitty gritty stuff

Reply:
I mean instead of manually testing extremes etc, there is always one thing we will miss

Reply:
For sure, im not super sure how to use those, any example you have in mind I could peek at?

Reply:
Done 👌 I added panic case checks for utility functions using prop-tests 👌